### PR TITLE
add release old tag

### PR DESCRIPTION
**Pull request by bearslyricattack**

### CODE REVIEW

File: controllers/devbox/internal/controller/devboxrelease_controller.go [hunk 1]
```diff
@@ -113,11 +113,16 @@ func (r *DevBoxReleaseReconciler) CreateReleaseTag(ctx context.Context, devboxRe
 	if err := r.Get(ctx, devboxInfo, devbox); err != nil {
 		return err
 	}
-	hostName, imageName, oldTag, err := r.GetHostAndImageAndTag(devbox)
+	hostName, imageName, oldTag, err := r.GetRegistryInfo(devbox)
 	if err != nil {
 		return err
 	}
 	logger.Info("Tagging image", "host", hostName, "image", imageName, "oldTag", oldTag, "newTag", devboxRelease.Spec.NewTag)
+	devboxRelease.Status.OriginalImage = imageName + oldTag
```
Comment (lingdie) [inline controllers/devbox/internal/controller/devboxrelease_controller.go:121]: miss `:`

**Review by lingdie — CHANGES_REQUESTED** [review]

File: controllers/devbox/api/v1alpha1/devboxrelease_types.go [hunk 2]
```diff
@@ -50,6 +50,8 @@ type DevBoxReleaseStatus struct {
 	// +kubebuilder:validation:Optional
 	// +kubebuilder:default=Pending
 	Phase DevboxReleasePhase `json:"phase"`
+	// +kubebuilder:validation:Optional
```
Comment (lingdie) [inline controllers/devbox/api/v1alpha1/devboxrelease_types.go:53]: add printcolumn for  `OriginalImage`

**Comment by lingdie** [comment]
make pre-deploy and add chanages to `deploy.yaml.tmpl`

**Comment by bearslyricattack** [comment]
> make pre-deploy and add chanages to `deploy.yaml.tmpl`

already do this，no more different.

**Review by lingdie — APPROVED** [review]

**Pull request merged**